### PR TITLE
Add missing env params for data importer and pgm/license scoring

### PR DESCRIPTION
**Pull request by miteshvp**
Fixes https://github.com/fabric8-analytics/fabric8-analytics-server/issues/90

### CODE REVIEW

**Comment by tisnik** [comment]
[test this]

**Comment by jpopelka** [comment]
[test this]

File: docker-compose.yml [hunk 1]
```diff
@@ -264,34 +282,36 @@ services:
...
-      AWS_BUCKET: "${USER}-bayesian-core-data"
+      AWS_EPV_BUCKET: "${USER}-bayesian-core-data"
+      AWS_PKG_BUCKET: "${USER}-bayesian-core-package-data"
       POSTGRESQL_USER: coreapi
       POSTGRESQL_PASSWORD: coreapi
       POSTGRESQL_DATABASE: coreapi
       PGBOUNCER_SERVICE_HOST: coreapi-pgbouncer
       BAYESIAN_PGBOUNCER_SERVICE_HOST: "coreapi-pgbouncer"
       BAYESIAN_PGBOUNCER_SERVICE_PORT: 5432
+      NUMBER_WORKER_PROCESS: 1
```
Comment (jpopelka) [inline docker-compose.yml]: Has already been added with 22826ad28

File: docker-compose.yml [hunk 2]
```diff
@@ -264,34 +282,36 @@ services:
...
+#    image: registry.devshift.net/bayesian/cvedb-s3-dump
+#    restart: on-failure
+#    network_mode: bridge
+#    environment:
+#      AWS_ACCESS_KEY_ID: "GNV3SAHAHA3DOT99GQII"
+#      AWS_SECRET_ACCESS_KEY: "ZmvMwngonaDK5ymlCd6ptaalDdJsCn3aSSxASPaZ"
+#      DEPLOYMENT_PREFIX: "${USER}"
+#      S3_ENDPOINT_URL: "http://coreapi-s3:33000"
+#    links:
+#      - minio-s3
```
Comment (jpopelka) [inline docker-compose.yml]: What's the reason for commenting out the `cvedb-s3-dump` service ?

File: openshift/cloud-deploy/secrets-template-aws.yaml [hunk 3]
```diff
@@ -9,57 +9,57 @@ kind: Template
...
+#  data:
+#    jwt-pubkey: |
+#      LS0tLS1CRUdJTiBQVUJMSUMgS0VZLS0tLS0KTUlJQklqQU5CZ2txaGtpRzl3MEJBUUVGQUFPQ0FR
+#      OEFNSUlCQ2dLQ0FRRUFzNzNJQlRvMHJYMm05YkdKR09GcQpOdEQyWGlONlkzbUxqWURuQklMUkhW
+#      UTNmeVpuVHkvcFJDK2FLUWwvckZiSmJ2MGNYSC9XZHFPVUd2NG81Y3NWCmNhUjdDUFdQUE5KZzRS
+#      cmtndHJKR0FZNVp4dTBBNFNmbHlVSTZSTW54YnJsZVAvMytiSGpTNVc0eEZVdFgvdUwKOFVtK3ds
+#      d3VSMjZ0UGVRQWU1SHlHTm8vbW1ETnpxb2hRR1ZjYTg5cUtmL0hGbkttWUxlTWNhV1FBSC9vMEtT
+#      Swp5WlZFS2xHNjg5eTNLMFRxNlhZQkMrU2tPSXJzV2NnNzFaclg2YXptOERQQmE2L2hTY2s2MTlI
+#      K0lMZTdWd2pYCnBaNHNTNXNMbzEwRTBzU0hOWmI1N284TXBHVFBCSFFQZ0tOTm5IR2xUbjJNeXpt
+#      TFBabThPQ3IrS3VGdG1DeE8KTXdJREFRQUIKLS0tLS1FTkQgUFVCTElDIEtFWS0tLS0t
```
Comment (jpopelka) [inline openshift/cloud-deploy/secrets-template-aws.yaml]: I don't understand whole this `secrets-template-aws.yaml` change. Can you shed some light on it please ?

File: docker-compose.yml [hunk 4]
```diff
@@ -196,8 +204,17 @@ services:
...
+      OUTLIER_THRESHOLD: 0.6
+      UNKNOWN_PACKAGES_THRESHOLD: 0.3
+      PGM_SERVICE_HOST: "kronos-stack-analysis.dev.rdu2c.fabric8.io"
+      LICENSE_SERVICE_HOST: "stack-license-stack-license.dev.rdu2c.fabric8.io"
+      PGM_SERVICE_PORT: "80"
+      LICENSE_SERVICE_PORT: "80"
   anitya-server:
-    image: registry.devshift.net/slavek/anitya-server
+    #image: registry.devshift.net/slavek/anitya-server
+    image: slavek/anitya-server
```
Comment (jpopelka) [inline docker-compose.yml]: Any reason for reverting fd6c5c15 ?

Comment (miteshvp) [inline docker-compose.yml]: @jpopelka, I have no idea how that went through. Must be a merge issue. I will likely close this against a new PR. Thanks.

**Pull request closed without merging**